firmware/console/status_loop.cpp

```diff
@@ -424,13 +427,15 @@ static void updateFuelCorrections() {
 }
 
 static void updateFuelResults() {
+	#ifdef MODULE_TRIP_ODO
```

Comment:
nit: `ifdef` vs `if` -- there are ~10x more `if` vs `ifdef` already; do we care?  my brain just keeps saying `clang-format`  :/

Reply:
they do different things - ifdef happens if the macro merely exists, and if happens if the thing is truthy

Reply:
it's simpler to define the symbol when the module is included, instead of having to define all not-loaded modules to 0, then re-define some to 1 if loaded